core/src/mindustry/ClientLauncher.java

```diff
@@ -182,6 +185,14 @@ public void exit(){
 
     @Override
     public void init(){
```

Comment:
This shouldn't be in `init()` - all initialization logic is in `setup()`.

Reply:
Done